capa/features/extractors/ida/helpers.py

```diff
@@ -26,11 +27,17 @@ def find_byte_sequence(start: int, end: int, seq: bytes) -> Iterator[int]:
         end: max virtual address
         seq: bytes to search e.g. b"\x01\x03"
     """
+    patterns = ida_bytes.compiled_binpat_vec_t()
+    encoding = ida_nalt.get_default_encoding_idx(ida_nalt.BPU_1B)
```

Comment:
`find_byte_sequence` may be called many times. Can we move the initialization of `patterns` and `encoding` to the beginning of the file at the global scope so they are only called once? If so, while making this change let's update their names to be more descriptive, e.g. `IDA_BYTES_PATTERNS` and `IDA_NALT_ENCODING`.

Reply:
As requested, I have declared the global variables for reuse in 78ea732ba91ed5106b3b6e3cf2b8a9d569af2f6c.

Reply:
Please confirm that both of these can be reused without needing to reinitialize before each use.